code-challenges.js

```diff
@@ -1,39 +1,74 @@
 
-// Please read all questions thoroughly!
-// If you get stuck, please leave comments to help us understand your thought process.
+// // Please read all questions thoroughly!
+// // If you get stuck, please leave comments to help us understand your thought process.
 
-// --------------------1) Create a statement that determines if a temperature is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit.
+// // --------------------1) Create a statement that determines if a temperature is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit.
 
-// Create the code and test each of the variables provided. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
+// // Create the code and test each of the variables provided. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
+//Function can be created using simple if statements.. If above or equal to 212, its above boiling. if below 42 its below boiling, else its at boiling.
 
-var temp = 42
+// var temp = 210
 // var temp = 350
-// var temp = 212
+var temp = 212
 
+if (temp > 211){
+    console.log(`${temp} is above boiling point`);
+}else if(temp < 43){
+    console.log(`${temp} is below boiling point`)
+}else{
+    console.log(`${temp} is at boiling point`)
+}
 
-// --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
+// // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
+//Can combine two arrays using the .concat() function. From there, length can be determined using .length
 
-var myNumbers1 = [3, 7, 0, 36, -9]
+var myNumbers1 = [3, 7, 0, 36, -9, 7]
 var myNumbers2 = [8, -7, 42, 9, 13]
 
+console.log(myNumbers1.concat(myNumbers2).length)
 
-// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ohcE"
+// // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ohcE"
+//.reverse() does not work on strings. Need to convert currentCohort into an array, and reverse that array. From there, other built in functions can be used to combine, and remove extra punctuations
 
 var currentCohort = "Echo 2021"
+var revCohort = []
 
+for (let i = 0; i < currentCohort.length; i++){
+    revCohort[i] = currentCohort[i]
+}
 
-// --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
+revCohort.reverse()
+
+console.log(revCohort.join(""))
+
```

Comment:
I like it! This absolutely works. However there is a method that will make this a little easier for you. Try `.split()`!